roles/ceph-common/defaults/main.yml

```diff
@@ -8,7 +8,7 @@ ceph:
   #  - sdd
   #bcache_ssd_device: sdg 
 
-  stable_release: hammer
+  stable_release: jewel
```

Comment:
The logic that uses this variable will need to change as with Jewel we will be using pack version.